CHANGELOG.d/breaking_switch-to-es-modules.md

```diff
@@ -0,0 +1,28 @@
+* Switch from Common JS to ES modules
+
+  Previously, Purescript used Common JS for FFI declarations.
+
+  Before, FFI was declared like this...
+
+  ```javascript
+  const mymodule = require('mymodule')
+
+  exports.myvar = mymodule.myvar
+  ```
+
+  ...and will be changed to this...
+
+  ```javascript
+  import * as M from 'mymodule';
+  export const myvar = M.myvar
+  ```
+  ...or using the short version...
+
+  ```javascript
+  export { myvar } from 'mymodule';
+  ```
+
+* FFI is annotated with `/* #__PURE__ */` so that bundlers can perform DCE
+* If CJS is detected a `Warning` is emitted
+* The current LTS Node.js version `12` is now the required minimum version
+* `purs bundle` has been rudimentarily updated but will be removed in a subsequent PR
```

Comment:
Does this line need to be included? After all, once we remove the bundler the combined changelog will reflect both of these changes.

Reply:
Oh, I was thinking that line would be removed in the PR that removes it. Each of these changelog entry files can be edited multiple times and each PR that edits it will be included in the list of PRs for that entry.

Reply:
Ah, I see. In that case carry on :)